src/rust/engine/src/lib.rs

```diff
@@ -607,6 +601,8 @@ pub extern "C" fn rule_subgraph_visualize(
   })
 }
 
+// TODO: This produces "thread panicked while processing panic. aborting." on my linux laptop,
+// requiring me to set RUST_BACKTRACE=1 (or "full") to get the panic message.
```

Comment:
This is #4884 I think... could link to it maybe.

Reply:
Done!